### DISCUSSION TITLE
Getting response in json using script in beta 9.3.0

### DISCUSSION THREAD

**Discussion by namlai000** [Kong/insomnia#7456] [Help]
i’m trying to get the response to json but unable in the beta version 9.3.0, im unable to do so, not sure if i wrote it with error, anyone has document or help me in this matter, thank you
![IMG_0871](https://github.com/Kong/insomnia/assets/16819440/9dc3dcf1-7d1c-44ea-b1b7-878862efc31e)

**Comment by ihexxa** [comment]
Hello @namlai000, have you created and selected a "sub" environment for persisting environment values? Currently if there is no environment selected (in the left bar), environment values persisting has no effect.

**Reply by namlai000** [reply]
okay so if i add the environment and add variables it will work?
i'm not sure if insomnia.response.json() is working or not but i'll try

**Reply by ihexxa** [reply]
Btw for debugging, you could also try console.log to print the json body, outputs will be in the timeline.
